lib/presentation/mixins/contacts_view_controller_mixin.dart

```diff
@@ -40,7 +43,7 @@ mixin class ContactsViewControllerMixin {
       getIt.get<SearchRecentChatInteractor>();
 
   ValueNotifier<WarningContactsBannerState> warningBannerNotifier =
-      ValueNotifier(WarningContactsBannerState.hide);
+      ValueNotifier(WarningContactsBannerState.display);
```

Comment:
- Why do you change variable? 
- Side effect from this:

https://github.com/user-attachments/assets/0c3c78c0-0a2a-4dd6-ad78-2f0694852545